recipe/meta.yaml

```diff
@@ -1,7 +1,7 @@
 {% set name = "coiled-runtime" %}
-{% set version = "0.2.0" + environ.get("VERSION_SUFFIX", '') %}
-{% set dask_version = environ.get("DASK_VERSION", "2022.6.0") %}
-{% set distributed_version = environ.get("DISTRIBUTED_VERSION", "2022.6.0") %}
+{% set version = "0.3.0" + environ.get("VERSION_SUFFIX", '') %}
```

Comment:
Are we releasing 0.2.0 or 0.3.0?

Reply:
This is for the nightlies, we usually put it one more up than the release it'll happen.